src/bots.json

```diff
@@ -865,6 +865,15 @@
         "Podcastics"
       ]
     },
+    {
+      "name": "Podcatcher",
+      "pattern": "^Podcatcher Bot 1.0",
+      "examples": [
+        "Podcatcher 0.1",
```

Comment:
This line was caught by the automated tests in this repo.  Is this an app string or a bot string?

Reply:
I misunderstood the purpose of the examples -- I thought they were also meant to show historical uses, which that one would've been an example of an older bot user agent. The bot now always uses "Podcatcher Bot 1.0", though, so I'll drop the example.